axonn/axonn.py

```diff
@@ -137,6 +137,11 @@ def init(
     config.inter_layer_parallel_rank = comm_handle.inter_layer_parallel_rank
     config.data_parallel_rank = comm_handle.data_parallel_rank
     config.intra_layer_parallel_rank = comm_handle.intra_layer_parallel_rank
+    config.intra_layer_depth_parallel_rank = comm_handle.intra_layer_depth_parallel_rank
+    config.intra_layer_row_parallel_rank = comm_handle.intra_layer_row_parallel_rank
+    config.intra_layer_column_parallel_rank = (
```

Comment:
Just curious on why the different formatting?

Reply:
automated by black